awswrangler/catalog/_utils.py

```diff
@@ -142,7 +164,10 @@ def sanitize_dataframe_columns_names(df: pd.DataFrame) -> pd.DataFrame:
     ----------
     df : pandas.DataFrame
         Original Pandas DataFrame.
-
+    handle_dup_cols : str, optional
```

Comment:
Nice! Nitpick - can we rename it to `handle_duplicate_columns`?

Reply:
For sure.